### DISCUSSION TITLE
Does OpenMPF have GPU support?

### DISCUSSION THREAD

**Discussion by jrobble** [openmpf/openmpf#1538] [Q&A] [answered]
We have some NVIDIA hardware we want to use.

**Comment by brosenberg42** [comment, marked as answer]
Currently the [OcvYoloDetection component](https://github.com/openmpf/openmpf-components/tree/master/cpp/OcvYoloDetection) and the [OcvDnnDetection component](https://github.com/openmpf/openmpf-components/tree/master/cpp/OcvDnnDetection) support CUDA. Our C++ base Docker images come with CUDA support. If a component developer has an algorithm that is capable of running on a GPU our base images will allow them to utilize one. Specifically, they include a build of OpenCV with CUDA enabled and have the necessary CUDA libraries installed.
